python/custreamz/custreamz/tests/test_kafka.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import confluent_kafka as ck
+import pytest
+
+from cudf.tests.utils import assert_eq
+
+from custreamz import kafka
+
+
+@pytest.fixture(scope="module")
+def client():
+    kafka_configs = {
+        "metadata.broker.list": "localhost:9092",
+        "enable.partition.eof": "true",
+        "group.id": "groupid",
+        "auto.offset.reset": "earliest",
+        "enable.auto.commit": "false",
+    }
+    return kafka.Consumer(kafka_configs)
```

Comment:
Move this into a `conftest.py` similar to https://github.com/rapidsai/cudf/blob/branch-0.15/python/cudf/cudf/tests/conftest.py and wrap it in a try except, where if it fails to create the client because there's no kafka cluster running then it does a `pytest.skip` instead. This way every test will be automatically skipped if there's no kafka cluster.